tests/testthat/test-class-Megastudy.R

```diff
@@ -293,9 +334,9 @@ test_that("imputeZeroes method is sane", {
   expect_equal(nrow(imputedDT[imputedDT$sample.specimen_count == 0]), 0)
```

Comment:
Does `imputedDT` have a column `sample.specimen_count` if we didn't ask for it?
If the column doesn't exist, does line 334 return zero and pass the test anyway?

Oh, I see that we kind of half-asked for it! (empty plotReference)

Reply:
if the column doesnt exist, the test wont pass. in this case i know the column was there bc its in mComplete@data, and we are no longer removing columns based on their presence in the VariableMetadataList.